addons/mrp_repair/models/mrp_repair.py

```diff
@@ -0,0 +1,510 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models, _
+import openerp.addons.decimal_precision as dp
+from openerp.exceptions import UserError
```

Comment:
Use ***odoo*** instead of ***openerp***
```
from odoo import api, fields, models, _
import odoo.addons.decimal_precision as dp
from odoo.exceptions import UserError
```